test/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "node/no-unsupported-features": "off"
```

Comment:
Could you add the changes I had made in https://github.com/okonet/lint-staged/pull/273/files#diff-1846b85e9e313f50dba02d1d9666c8bc

Reply:
`"import/no-extraneous-dependencies": [2, { "devDependencies": true }]` is unnecessary BTW

we already derived it from Airbnb's config & okonet`s config